foundry.toml

```diff
@@ -1,6 +1,6 @@
 [profile.default]
 solc-version = "0.8.26"
-evm_version = "paris"
+evm_version = "cancun"
```

Comment:
why?

we cannot make this change because going above paris, will enable push0 opcode which is not compatible on many evm chains

Reply:
Looked into this more, they have 2 versions of their `TransferValidator` contract.
The first version is on Cancun and the second version is on Paris

**1st Version**
https://etherscan.io/address/0x721c0078c2328597ca70f5451fff5a7b38d4e947#code

**2nd Version**
https://etherscan.io/address/0x721C00182a990771244d7A71B9FA2ea789A3b433#code

In the first version, it uses the `validateTransfer` function to validate the transfers
but the second version uses `applyCollectionTransferPolicy` function to validate the transfers

The implementation that they made uses `validateTransfer` meaning that they implemented the extension with using the first version in mind

This is mirrored in the tests as they make an explicit check to see if it's on the cancun chain.
https://github.com/thirdweb-dev/modular-contracts/blob/fc68e567ff86b174d6137247d61629d1caa111bc/test/module/royalty/RoyaltyERC721.t.sol#L260

With this in mind, there are 2 actionables:
1. ask Limit Break team if there is a reason why they are using the first version over the second and if it is okay to use the second version in our modules
2. Update the royalty modules and tests to use the second version instead